src/loaders/styleguide-loader.ts

```diff
@@ -27,6 +30,9 @@ const CLIENT_CONFIG_OPTIONS = [
 	'version',
 ];
 
+const StylesVariableName = '__rsgStyles';
```

Comment:
Let's not use class name convention for constants.

```suggestion
const STYLE_VARIABLE_NAME = '__rsgStyles';
```

Reply:
agreed